tests/test_uri.py

```diff
@@ -483,7 +566,6 @@ def testTemporary(self):
             self.assertFalse(tmp.exists(), f"uri: {tmp}")
             tmp.write(b"abcd")
             self.assertTrue(tmp.exists(), f"uri: {tmp}")
-            self.assertTrue(tmp.isTemporary)
```

Comment:
Why was this deleted?

Reply:
I think this was another casualty of my aborted attempt to add directory support to `temporary_uri`.  Will restore.